lib/Models/ResultPendingCatalogItem.js

```diff
@@ -25,12 +25,12 @@ var AsyncFunctionResultCatalogItem = function(terria) {
     knockout.track(this, ['loadingMessage', 'shortReport']);
```

Comment:
`shortReport` is a property on `CatalogMember` now, so yeah, we can get rid of it here.